services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -855,20 +859,56 @@ class FlowCrudSpec extends HealthCheckSpecification {
             }.size() == broughtDownPorts.size() * 2
         }
 
-        and: "Make the first selected path unstable by bringing port down/up"
+        and: "First path is unstable (due to bringing port down/up)"
         // after bringing port down/up, the isl will be marked as unstable by updating the 'time_unstable' field in DB
         def islToBreak = pathHelper.getInvolvedIsls(firstPath).first()
         antiflap.portDown(islToBreak.srcSwitch.dpId, islToBreak.srcPort)
         Wrappers.wait(WAIT_OFFSET) { assert islUtils.getIslInfo(islToBreak).get().state == FAILED }
         antiflap.portUp(islToBreak.srcSwitch.dpId, islToBreak.srcPort)
         Wrappers.wait(WAIT_OFFSET) { assert islUtils.getIslInfo(islToBreak).get().state == DISCOVERED }
 
+        and: "Total cost of second path more preferable than the first one"
+        def involvedIslsInFirstPath = pathHelper.getInvolvedIsls(firstPath)
+        def totalCostFirstPath = involvedIslsInFirstPath.sum {
```

Comment:
Let's not call it 'total cost'. This associates with sum of all isls costs. Let's call it 'unstable cost' or sth like that. Since the unstable isl cost is an implicit mechanic which is not visible when looking at actual isl costs